lib/internal/bootstrap_node.js

```diff
@@ -188,6 +188,7 @@
   }
 
   function setupGlobalVariables() {
+    global[Symbol.toStringTag] = 'global';
```

Comment:
I'd suggest using `Object.defineProperty` here, so that you can make it nonenumerable, and preferably nonwritable as well.